app/javascript/components/GroupEdit.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div
+    id="EditMembers"
+  >
+    <div class="max-w-md">
+      <div class="text-blue-800 m-2 mb-5">
+        Miembros actuales
+      </div>
+      <div v-if="error.length">
+        {{ error }}
+      </div>
+      <div class="m-2">
+        <input
+          class="ml-3"
+          type="checkbox"
+        >
+        <span class="text-blue-800 ml-3">
+          {{ actualMembers.length }} miembros
+        </span>
+        <button
+          @click="deleteMembers"
+          class="float-right text-red-600 underline"
+        >
+          Eliminar
+        </button>
+      </div>
+
+      <div
+        v-for="member in actualMembers"
+        :key="member.id"
+        class="flex items-center h-16 bg-opacity-75 rounded-lg m-2"
+      >
+        <Group
+          :member="member"
+          :select="false"
+          :edit="true"
+          @check-member="checkMember"
+          @uncheck-member="unCheckMember"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+import Group from '../components/Group.vue';
+
+export default {
+  components: {
+    Group,
+  },
+  props: {
+    group: {
+      type: Object,
+      required: true,
+    },
+    actualMembers: {
+      type: Array,
+      required: true,
+    },
+  },
+  methods: {
+    async deleteMembers() {
+      try {
+        let ids = this.actualMembers.filter((member) => !this.selectedMembers.includes(member.id));
+        ids = ids.map((member) => member.id);
+        await axios({
+          method: 'put',
+          url: `/api/v1/teams/${this.group.id}`,
+          headers: this.$store.getters.header,
+          data: {
+            // eslint-disable-next-line camelcase
+            user_ids: [... ids],
```

Comment:
esto debería funcionar pasando los ids no más o no :thinking: ?
```suggestion
            user_ids:  ids,
```

Reply:
toda la razon